bokehjs/src/lib/models/axes/axis.ts

```diff
@@ -341,6 +341,12 @@ export class AxisView extends GuideRendererView {
   }
 
   get offsets(): [number, number] {
+    // If we have a fixed_position then we should respect that exactly and
+    // not apply any offsets (https://github.com/bokeh/bokeh/issues/8552)
+    if (this.fixed_location != null) {
```

Comment:
This should be `this.model.fixed_location`. I will fix that independently, because I already have rebased commits that would overwrite this anyway.